docs/src/main/asciidoc/elasticsearch.adoc

```diff
@@ -6,14 +6,24 @@ https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 = Connecting to an Elasticsearch cluster
 include::_attributes.adoc[]
 :categories: data
-:summary: This guide covers how to use an Elasticsearch cluster using the low level or high level REST clients.
+:summary: This guide covers how to query an Elasticsearch cluster using the low level REST client or the Elasticsearch Java client.
 
 Elasticsearch is a well known full text search engine and NoSQL datastore.
 
-In this guide, we will see how you can get your REST services to use an Elasticsearch cluster.
+In this guide, we will see how you can get your REST services to query an Elasticsearch cluster.
 
-Quarkus provides two ways of accessing Elasticsearch: via the lower level `RestClient` or via the `RestHighLevelClient` we will call them
-the low level and the high level clients.
+Quarkus provides two ways of accessing Elasticsearch:
+
+* The lower level REST Client
+* The Elasticsearch Java client
+
+[WARNING]
+====
+The Quarkus extension for the high level REST Client is deprecated and will be removed in a future version,
```

Comment:
Maybe clarify that the high level REST client is not the same as the "Java client" mentioned above? It could be confusing.

```suggestion
A third Quarkus extension for the "high level REST Client" exists, but is deprecated and will be removed in a future version,
```